zproject/computed_settings.py

```diff
@@ -382,6 +382,8 @@ def get_dirs(self) -> List[PosixPath]:
 # other domains, but we use this list as an extra precaution.
 RATE_LIMITING_DOMAINS_FOR_TORNADO = ["api_by_user"]
 
+# A note regarding incoming emails limits can be found here:
```

Comment:
I liked the previous wording on this comment better -- I'm not sure why you changed it between versions.